app/services/abstract_manual_document_service_registry.rb

```diff
@@ -125,9 +125,7 @@ def publishing_api_draft_manual_document_exporter
       ).call
 
       SectionPublishingAPIExporter.new(
-        publishing_api_v2.method(:put_content),
```

Comment:
You mean we're no longer passing methods around forcing us to use `.call` everywhere and making things harder to understand?

👏 👏 👏 
➕ 💯